src/components/MovieDetails/MovieDetails.js

```diff
@@ -2,21 +2,16 @@ import React, { Component } from 'react';
 import { getRatings } from '../../actions';
 import { connect } from 'react-redux';
 
-import { apiFetchRatings, apiPostNewRating } from '../../apiCalls/apiCalls';
+import { apiFetchRatings, apiPostNewRating, apiDeleteRating } from '../../apiCalls/apiCalls';
 
 class MovieDetails extends Component {
   constructor() {
     super();
     this.state = {
-      userRating: null
+      userRating: 1
```

Comment:
I need to change the default value to 1 instead of null.  If a user is logged in and remove a rating and rating it again without changing the null value, then the fetch will not run since it is passing in a value of null.  So, by default, the value always start at 1.